hw/hw17/html_render.py

```diff
@@ -5,7 +5,109 @@
 
 """
 
+
 # The start of it all:
 # Fill it all in here.
 class Element(object):
-    pass
+
+    BASE_INDENT = "    "
+
+    def __init__(self, tag="", content=None, oneLineFlag=False,
+                 selfCloseFlag=False, **kwargs):
+        self.tag = tag
+        self.content = []
+        self.append(content)
```

Comment:
This should probably be self.content.append(content) but this appears to work as well. Interesting